lib/payload.json

```diff
@@ -99,6 +104,62 @@
         "air": {
           "type": "object",
           "properties": {
+            "indoor": {
```

Comment:
I agree indoor/outdoor is super useful. And we probably should have gone with this from the start.

Now we have temperature, indoor temperature and outdoor temperature. I think instead we should add a field to indicate whether `air` was measured indoor or outdoor. This way we can reuse the same fields. Sensors able to measure indoor and outdoor concurrently can record two distinct measurements.